web-ui/src/main/resources/catalog/components/common/map/print/PrintDirective.js

```diff
@@ -494,7 +494,7 @@
             var tileGrid = source.getTileGrid();
             angular.extend(enc, {
               type: 'WMTS',
-              baseURL: location.protocol + '//wmts.geo.admin.ch', // FIXME
+              baseURL: '//wmts.geo.admin.ch', // FIXME
```

Comment:
I don't known what has to be fixed in this line. I think it is more related to the hardcoded domain than protocol.